modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/ApplicationAPI.macro

```diff
@@ -32,6 +34,13 @@ definition {
 		Variables.assertDefined(parameterList = "${baseURL},${status},${title}");
 
 		var curl = ApplicationAPI._curlAPIApplication(virtualHost = ${virtualHost});
+
```

Comment:
Could you take a look at it? I'm not sure where can we use externalReferenceCodeApp, it only appears in this if the function